opc/resource_storage_object_test.go

```diff
@@ -0,0 +1,173 @@
+package opc
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-oracle-terraform/storage"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const _TestStorageObjectPath = "test-fixtures"
+
+func TestAccOPCStorageObject_contentSource(t *testing.T) {
```

Comment:
Given fields like `content_type` are Optional but always supplied in the test - it'd be good to add a "basic" test with only the required fields set if possible?